GPy/util/choleskies.py

```diff
@@ -73,34 +80,42 @@ def _backprop_gradient_pure(dL, L):
         for j in range(k + 1, N):
             dL_dK[j, k] /= L[k, k]
             dL_dK[k, k] -= L[j, k] * dL_dK[j, k]
-        dL_dK[k, k] /= (2 * L[k, k])
+        dL_dK[k, k] /= 2 * L[k, k]
     return dL_dK
 
+
 def triang_to_cov(L):
-    return np.dstack([np.dot(L[:,:,i], L[:,:,i].T) for i in range(L.shape[-1])])
+    return np.dstack([np.dot(L[:, :, i], L[:, :, i].T) for i in range(L.shape[-1])])
+
 
 def multiple_dpotri(Ls):
-    return np.array([linalg.dpotri(np.asfortranarray(Ls[i]), lower=1)[0] for i in range(Ls.shape[0])])
+    return np.array(
+        [
+            linalg.dpotri(np.asfortranarray(Ls[i]), lower=1)[0]
+            for i in range(Ls.shape[0])
+        ]
+    )
+
 
 def indexes_to_fix_for_low_rank(rank, size):
     """
     Work out which indexes of the flatteneed array should be fixed if we want
     the cholesky to represent a low rank matrix
     """
-    #first we'll work out what to keep, and the do the set difference.
+    # first we'll work out what to keep, and the do the set difference.
 
-    #here are the indexes of the first column, which are the triangular numbers
+    # here are the indexes of the first column, which are the triangular numbers
     n = np.arange(size)
-    triangulars = (n**2 + n) / 2
+    triangulars = (n ** 2 + n) / 2
     keep = []
     for i in range(rank):
         keep.append(triangulars[i:] + i)
-    #add the diagonal
-    keep.append(triangulars[1:]-1)
-    keep.append((size**2 + size)/2 -1)# the very last element
+    # add the diagonal
+    keep.append(triangulars[1:] - 1)
+    keep.append((size ** 2 + size) / 2 - 1)  # the very last element
```

Comment:
I like `black` because it has style opinions so I don't have to 😅 It's not a panacea, but it _is_ identical everywhere. If this stylistic imposition is cool, please resolve; otherwise we should discuss.